### ISSUE TITLE
linking to lgomp when compiling with Intel compilers

### ISSUE THREAD

**Issue by solomonik** [devinamatthews/aquarius#14]
When configuring on Edison with an Intel environment, AQ includes the -lgomp link flag, yielding to both Intel and GNU OpenMP symbols being linked in, and causing errors when running with threads.  We dealt with a similar problem before, I don't remember how/whether it was resolved.

**Comment by devinamatthews** [comment]
Try configuring with FC=ifort.

**Comment by solomonik** [comment]
I tried rebuilding with FC=ftn, but there is still -lgomp in FCLIBS in the resulting Makefile.  It would be nice to ensure -lgomp does not get into FCLIBS when compiling with Intel CXX.  It results in a runtime memory corruption when running with threads, which is a frustrating bug to encounter/debug since it looks like a logical error in some threaded code region in CTF or AQ.  I've spent a bunch of time tracking it down twice now, did the commit that fixed it last time (if there was one, I am not 100% sure) get undone when you reverted recently?
